pages/en/learn/command-line/how-to-read-environment-variables-from-nodejs.md

```diff
@@ -47,3 +47,5 @@ process.env.NODE_ENV; // "development"
 ```
 
 > You can also run your js file with `node -r dotenv/config index.js` command if you don't want to import the package in your code.
+
+> Note: Nodejs 20 and above have **experimental** support for the .env file. You can read more about it [here](https://nodejs.org/dist/latest-v20.x/docs/api/cli.html#--env-fileconfig).
```

Comment:
I feel that we shouldn't mention "Since Nodejs 20 and above", also this grammar feels awkward at best.

I think this guide should be updated to include a whole section of reading .env files. @nodejs/nodejs-website if anyone wants to write that, or @AugustinMauroy or @canerakdas (I can review the changes)

Also @bmuenzenmeyer can proof check the changes

Reply:
perhaps this `.env` file addition should be a separate issue - otherwise this PR will be unresolved for longer. I have some light edits in my review that I think address your comments here

Reply:
created https://github.com/nodejs/nodejs.org/issues/6140